### ISSUE TITLE
UnhandledPromiseRejectionWarning on propose-release crashes webpack

### ISSUE THREAD

**Issue by tevin** [getsentry/sentry-webpack-plugin#148]
When, `sentry-cli releases propose-version` fails, it throws the following error

```
(node:52729) UnhandledPromiseRejectionWarning: Error: Command failed: /Users/tevin/dev/umnatha-test/node_modules/@sentry/cli/sentry-cli releases propose-version
  INFO    2019-10-21 23:32:27.516651 +02:00 Loaded config from /Users/tevin/dev/umnatha-test/.sentryclirc
  DEBUG   2019-10-21 23:32:27.526384 +02:00 sentry-cli version: 1.48.0, platform: "darwin", architecture: "x86_64"
  INFO    2019-10-21 23:32:27.527283 +02:00 sentry-cli was invoked with the following command line: "/Users/tevin/dev/umnatha-test/node_modules/@sentry/cli/sentry-cli" "releases" "propose-version"
  DEBUG   2019-10-21 23:32:27.542115 +02:00 error: running update nagger
  DEBUG   2019-10-21 23:32:27.542442 +02:00 skipping update nagger because session is not attended
error: Could not automatically determine release name
  DEBUG   2019-10-21 23:32:27.560204 +02:00 client close; no transport to shut down  (from sentry)

    at ChildProcess.exithandler (child_process.js:294:12)
    at ChildProcess.emit (events.js:198:13)
    at maybeClose (internal/child_process.js:982:16)
    at Socket.stream.socket.on (internal/child_process.js:389:11)
    at Socket.emit (events.js:198:13)
    at Pipe._handle.close (net.js:606:12)
(node:52729) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
```

This is fine, since it is indeed a relevant warning/error.
However, downstream, it seems to crash the webpack process silently before the assets are copied to the `/dist` folder.

For example, on angular, while running `ng build`, the process exits at 
```
69% building 1155/1156 modules 1 active ...ack-plugin/src/sentry-webpack.module.js
```
with no error code (indicating a successful build)

I propose expected functionality could be `sentry-webpack.module.js` itself would throw a webpack error, or a adding a `.catch()` to the [`proposeVersion()`](https://github.com/getsentry/sentry-cli/blob/master/js/releases/index.js#L103) function call that stops the release from being created and source maps from being uploaded entirely (probably with a warning) but doesn't crash the webpack process.

For context, this is especially dangerous with Heroku. The process for setting up sentry is not always straightforward, because despite enabling runtime labs, it seems like you also have to set up [Deploy Hooks](https://blog.sentry.io/2017/05/15/heroku-commits) for the webpack plugin to work properly.
This means a build failure exiting with a successful return code would be deployed to production, despite there being no compiled assets in the `dist/` folder. 
Same thing applies for Heroku review apps, where runtime labs are disabled entirely. 

Thank you & let me know what you think :)

**Comment by kamilogorek** [comment]
I successfully reproduced it locally using angular-cli. I will patch it tomorrow morning.

![image](https://user-images.githubusercontent.com/1523305/68156300-d7395680-ff4b-11e9-98f7-3294833fd6e2.png)

**Comment by kamilogorek** [comment]
Released in 1.9.3. See docs for `errorHandler` how to change it to a warning.
